### ISSUE TITLE
Pipeline init usage bug

### ISSUE THREAD

**Issue by erno** [aws/copilot-cli#1152]
I followed the example in the pipeline init example inthe built in help, which says:

```Examples
  Create a pipeline for the services in your workspace.
  `$ copilot pipeline init \`
  ` --github-url https://github.com/gitHubUserName/myFrontendApp.git \`
  ` --github-access-token file://myGitHubToken \`
  ` --environments "stage,prod" \`
  ` --deploy`
```

But the pipeline errored, and i got it working by taking out the .git part from the url in copilot/pipeline.yml

edit: I was using the latest release (0.1.0).

**Comment by kohidave** [comment]
Ahh, thank you so much for reaching out! Yea - you're right, we want the GH URL w/o the `.git` at the end. The docs on the `pipeline init` page were wrong. I've gone ahead and updated them. I've also opened an issue to make `pipeline init` more flexible with the input you give it #1153 

Thanks for your feedback. Also, just as an FYI - you can omit the flags and if you're working in a git repo associated with that  GH repo, we'll auto detect it. 

Hope this helps!

**Comment by efekarakus** [comment]
This was released in [v0.3.0](https://github.com/aws/copilot-cli/releases/tag/v0.3.0)!
